Code/Framework/AzFramework/AzFramework/UnitTest/TestDebugDisplayRequests.cpp

```diff
@@ -58,8 +58,9 @@ namespace UnitTest
         m_points.push_back(tm.TransformPoint(AZ::Vector3(0.5f * width, 0.0f, 0.5f * height)));
     }
 
-    void TestDebugDisplayRequests::DrawQuad(float width, float height)
+    void TestDebugDisplayRequests::DrawQuad(float width, float height, bool drawShaded)
```

Comment:
nit: For new code please prefer `[[maybe_unused]]` in the parameter list as opposed to cast to void in the function body

Reply:
`void` is removed.

Reply:
Awesome thanks!